EssentialCSharp.Web/Services/SiteMappingDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace EssentialCSharp.Web.Services;
+
+public class SiteMappingDto
```

Comment:
nit: Some comments around the purpose of this `SiteMappingDTO` to distinguish it from `SiteMapping` could be helpful for the future since they fundamentally carry different data on them, but the names are similar enough to be confusing

Reply:
Good idea. I added one, but feel free to suggest any changes :)